packages/twenty-server/src/engine/metadata-modules/remote-server/remote-table/remote-table.service.ts

```diff
@@ -151,6 +151,15 @@ export class RemoteTableService {
       input.schema,
     );
 
+    // We only support remote tables with an id column for now.
+    const remoteTableIdColumn = remoteTableColumns.filter(
+      (column) => column.columnName === 'id',
+    )?.[0];
```

Comment:
You can use array.find

```suggestion
    const remoteTableIdColumn = remoteTableColumns.find(
      (column) => column.columnName === 'id',
    );
```